src/spectrecoff-cli/commands/Grid.fs

```diff
@@ -0,0 +1,66 @@
+﻿namespace SpectreCoff.Cli.Commands
+
+open System
+open Spectre.Console
+open Spectre.Console.Cli
+open SpectreCoff
+
+type GridSettings()  =
+    inherit CommandSettings()
+
+type GridExample() =
+    inherit Command<GridSettings>()
+    interface ICommandLimiter<GridSettings>
+
+    override _.Execute(_context, _settings) =
+        let player = "X"
+        let basket = "O"
+        let division = "|"
+
+        // the number of columns in the grid must match the number of each row
+        let basketballGame = createGridWithColumns(9)
+                               .AddRow("", player, "", player, division, player, "", player, "")
```

Comment:
@EluciusFTW 
Done, but not sure if I should still split them into submodules. Mostly because of naming :) With Chart that was easy.
Another doubt I have in that regard is about the `addRowTo...` methods. 
I know overloads can be simulated with passing a DU, but I am not sure this is better here. 
Anyway, take a look and see what you think.

Reply:
I'm not sure either. I do think I'd split them, but we can wait until we tackle the layout module. I feel something similar is going to happen there. So we might have a Layout module that has submodules for table, grid and layout, as all are concerned with layout, imo. And then a shared submodule with things like `Row`, etc. If there are submodules, we can require qualification, so the functions would be called `Grid.addRow` and` Table.addRow` instead of `addRowToTable` - or you could also right now make the add... private, and only access them as extension methods on `Grid` resp. `Table`, which are both called `addRow`!